web/dashboard.html

```diff
@@ -5,24 +5,33 @@
 <html>
 
 <head>
-    <title>Religion in Place Dashboard</title>
+    <title>Dashboard</title>
+    <link rel="shortcut icon" type="image/x-icon" href="./images/favicon/favicon.ico">
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <link rel="stylesheet" href="css/lr.css">
 </head>
 
 <body>
-    <!-- Is there a difference between the admin dashboard and the regular user dashboard -->
```

Comment:
Yes, and it does, I think that is an old note.

The reason the <lr-nav> has those tabs hard coded is so that there is never a moment where the tabs are missing and then pop in, causing the page to restructure which feels like a page bump.